src/Common/src/System/Net/Security/CertificateHelper.cs

```diff
@@ -0,0 +1,84 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Why is this part of System.Net.Security? Seems HTTP specific. The code is still duplicated in System.Net.Security AFAIK.